packages/protovalidate/src/planner.ts

```diff
@@ -102,7 +102,7 @@ export class Planner {
     const e = new EvalMany<ReflectMessage>();
     this.messageCache.set(message, e);
     if (!messageRules.disabled) {
-      e.add(this.fields(message.fields));
+      e.add(this.fields(messageRules, message.fields));
```

Comment:
For good style, let's pass only what's needed:

```suggestion
      e.add(this.fields(messageRules.oneof, message.fields));
```

Same as two lines below.